### PR TITLE
Adds a notable change regarding canary updating

### PR DESCRIPTION
**Pull request by stevsmit**
GH Issue at https://github.com/openshift/openshift-docs/issues/33497

Preview: https://deploy-preview-36810--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-9-release-notes.html#ocp-4-9-notable-technical-changes

@jiajliu please review

### CODE REVIEW

File: release_notes/ocp-4-9-release-notes.adoc [hunk 1]
```diff
@@ -555,6 +555,12 @@ To manage the cloud controller manager and cloud node manager deployments and li
 
 For more information, see the xref:../operators/operator-reference.adoc#cluster-cloud-controller-manager-operator_red-hat-operators[Cluster Cloud Controller Manager Operator] entry in the _Red Hat Operators reference_.
 
+[discrete]
+[id="ocp-4-9-canary-rollout-update-changes"]
+==== Performing a canary rollout update
+
+With {product-title} 4.9, the process to perform a canary rollout update has changed. For a detailed overview of this process, see xref:../updating/update-using-custom-machine-config-pools.html#update-using-custom-machine-config-pools-about_update-using-custom-machine-config-pools[About performing a canary rollout update].
```
Comment (bergerhoffer) [inline release_notes/ocp-4-9-release-notes.adoc:562]: `.html` -> `.adoc` in the xref suffix

**Review by bergerhoffer — COMMENTED** [review]
Just one minor thing, then LGTM!

**Comment by jiajliu** [comment]
@jianlinliu Could you take a look when you are available? Thanks!

File: release_notes/ocp-4-9-release-notes.adoc (hunk 1 above)
Comment (jianlinliu) [reply release_notes/ocp-4-9-release-notes.adoc:562]: > the process to perform a canary rollout update has changed

I think this is our 1st time to introduce canary rollout update in our official doc, but not change some existing ones, right?

**Comment by stevsmit** [comment]
@jiajliu @jianlinliu please re-review :)

**Comment by bergerhoffer** [comment]
LGTM, merging

**Pull request merged**